frontend/src/component/common/ToastRenderer/ToastRenderer.tsx

```diff
@@ -36,7 +35,7 @@ const ToastRenderer = () => {
                 right: 0,
                 left: 0,
                 margin: '0 auto',
-                maxWidth: '450px',
+                width: 'fit-content',
```

Comment:
is it what you were asking for in slack?

Reply:
No, not quite, but it's part of it 💁🏼 This just makes the toast be as wide as it has to instead of always the same width.